brianlong/www.validators.app
# 183730516 - add pythnet in views

#### What's this PR do?
PR applies the pythnet to views.

#### How should this be manually tested?
Things that having impact due to ticket (please test them):
- commision histories (index)
- stake account (index)
- ping things (index)
- check homepage if data is correct (switch between networks)

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183730516)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
